src/views/Dashboard/Customer/Home/RequestTest/AgreementDialog.vue

```diff
@@ -144,6 +145,16 @@ export default {
     },
 
     async submitServiceRequestStaking() {
+      const sufficientBalance = Number(this.amount) + Number(this.txWeight) <= Number(this.walletBalance)
```

Comment:
What do you want to get in this logic? it seems difficult to understand

Reply:
the case is `The insufficient balance pop up doesn't appear for cases where the amount of DBIO is there, but not enough for staking. So it's just loading after clicking staking`, so a condition is created before submitting to determine whether the balance is sufficient or not

Reply:
ok, so I think you need to add parentless `(Number(this.amount) + Number(this.txWeight)) <= Number(this.walletBalance)`

Reply:
okay i will update ASAP

Reply:
updated